File: server/order-service/src/main/java/com/sudo248/orderservice/controller/order/dto/SupplierInfoDto.java
package com.sudo248.orderservice.controller.order.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SupplierInfoDto implements Serializable {
    private String supplierId;
    private int ghnShopId;
    private String name;
    private String avatar;
    private String contactUrl;
    private AddressDto address;
    private float rate;
}

File: server/product-service/src/main/kotlin/com/mshop/productservice/dto/SupplierInfoDto.kt
package com.mshop.productservice.dto

data class SupplierInfoDto(
    val supplierId: String? = null,
    val ghnShopId: Int? = null,
//    val userId: String,
    val name: String,
    val avatar: String,
    val address: AddressDto,
    val contactUrl: String,
    val rate: Float
)